presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -215,16 +215,88 @@ The connector maps PrestoDB types to the corresponding MySQL types:
 
 No other types are supported.
 
+MySQL Connector Support
+-----------------------
+
+The MySQL connector allows querying and creating MySQL tables. Here are some examples of the SQL operations supported:
+
+CREATE TABLE
+^^^^^^^^^^^^
+
+Create a new MySQL table named ``page_views`` in the ``web`` schema:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views (
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    );
+
+INSERT INTO
+^^^^^^^^^^^
+
+Insert data into the ``page_views`` table:
+
+.. code-block:: sql
+
+    INSERT INTO mysql.web.page_views VALUES(1, 'https://example.com', current_date, 'country');
+
+CREATE TABLE AS SELECT
+^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new table ``page_views_new`` from an existing table ``page_views``:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views_new AS SELECT * FROM mysql.web.page_views;
+
+SELECT
+^^^^^^
+
+.. code-block:: sql
+
+    SELECT * FROM mysql.web.page_views;
+
+ALTER TABLE
+^^^^^^^^^^^
+
+.. code-block:: sql
+
+    ALTER TABLE mysql.web.page_views ADD COLUMN zipcode VARCHAR;
+    ALTER TABLE mysql.web.page_views RENAME COLUMN zipcode TO location;
+    ALTER TABLE mysql.web.page_views DROP COLUMN location;
+
+TRUNCATE
+^^^^^^^^
+
+Delete all of the data from the table ``page_views`` without dropping the table:
+
+.. code-block:: sql
+
+    TRUNCATE TABLE mysql.web.page_views;
+
+
 MySQL Connector Limitations
 ---------------------------
 
-The following SQL statements are not yet supported:
+The following SQL statements are not supported:
```

Comment:
Suggest alphabetizing this list following the order in [SQL Statement Syntax](http://prestodb.io/docs/0.286/sql.html), and following the [Code Style](https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md#code-style) recommendation in [CONTRIBUTING.md](https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md).